packages/lit-dev-content/site/docs/v3/components/decorators.md

```diff
@@ -77,6 +83,45 @@ import {customElement} from 'lit/decorators/custom-element.js';
 import {eventOptions} from 'lit/decorators/event-options.js';
 ```
 
+## Standard decorators { #standard-decorators }
+
+"Standard decorators" is the version of decorators that has reached Stage 3 consensus at TC39, the body that defines ECMAScript/JavaScript. The decorators specification has been fine-tuned with active VM implementor feedback to be as optimizable as possible.
```

Comment:
Is "active VM implementor feedback" too much detail?

Reply:
The whole sentence could also be dropped. If anything, we might want to say more like "At this stage, all semantics, syntax and API are completely described with no change expected barring critical findings from implementors."

Reply:
Good call. Changed to that

Reply:
Removed this since we repeat Stage 3 multiple times through the document.